app/views/contexts/list_context.py

```diff
@@ -95,12 +96,17 @@ def _build_list_items_context(
 
             yield list_item_context
 
-    def _get_item_title(self, summary_definition, list_item_id, is_primary):
+    def _get_item_title(
+        self,
+        summary_definition: Mapping[str, Any],
+        list_item_id: Optional[str],
+        is_primary: bool,
+    ) -> Any:
```

Comment:
This should be dict. Explicitly type if needed.

Reply:
Marked self._placeholder_renderer.render return type as dict. Still rendered_summary["item_title"] += lazy_gettext(" (You)") will return Any

Reply:
👍